Automattic/jetpack
# Search: temporary fix for Jetpack 10.8-a.9 submenu item

#### Changes proposed in this Pull Request:

Prevent the nested `Search` submenu item from showing on sites without eligible Search plans. Currently sites without Search plans will see the `Search` submenu item, but when visiting that page users can't interact with it so it's confusing.

This is intended to be a temp fix so we can get the Atomic release out until Search team can have a further look.

#### Jetpack product discussion

- Internal: p1648147013140099-slack-C82FZ5T4G

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* Pre-patching, on a JP connected site with no Search plan, running bleeding edge or RC, verify you see the Search submenu item nested under Jetpack. Visit the Search page - the toggles won't be able to be used.
* Patch with these changes, the Search submenu item should be hidden.
* Add a paid Search plan, and the Search submenu should be visible & functional.
